src/components/UI/Modal/Modal/Modal.js

```diff
@@ -31,17 +31,23 @@ export const Modal = ({
   show,
   type = ModalType.INFO,
   size = ModalSize.MEDIUM,
+  children,
```

Comment:
Should be place last

Reply:
Again, children should be the last prop.
Second, don't pass the `hideModal` as a prop from outside, you can use the `ModalWrapper` to pass it (we're using `useHideModal` inside the  ModalWrapper)